src/import.ml

```diff
@@ -69,3 +74,19 @@ let message kind fmt =
       let (_ : unit Promise.t) = k x in
       ())
     fmt
+
+module Log : sig
+  type field
+
+  val field : _ -> field
+end = struct
+  type field
+
+  external field : _ -> field = "%identity"
+end
+
+let log fmt = Printf.ksprintf (fun x -> Js.Console.log x) fmt
+
+let logJson msg (fields : (string * Log.field) list) =
+  let json = Js.Dict.fromList fields in
+  Js.Console.log2 msg json
```

Comment:
Would it be possible to use the native vscode output pane for logs? It seems like the more appropriate place for logs.

https://code.visualstudio.com/api/references/vscode-api#window.createOutputChannel
https://code.visualstudio.com/api/references/vscode-api#OutputChannel

Reply:
Isn't that where the console log statements go already? Or do you think we should have a dedicated output channel?

Reply:
Yes, currently all log statements go to the developer console. I think it would be useful to have a dedicated output channel as it would make it more user-friendly.